config/crd/bases/batch.volcano.sh_jobs.yaml

```diff
@@ -115,6 +115,11 @@ spec:
                 items:
                   description: TaskSpec specifies the task specification of Job.
                   properties:
+                    minAvailable:
+                      description: The minimal available pods to run for this Task
+                        Defaults to the task replicas
```

Comment:
If the default value is task replicas, does it mean that the minAvailable to the job is the sum of the minAvailable for all tasks?

Reply:
yes

Reply:
> yes

So if a user doesn't set task minAvailable but set job minavailable, which keeps the original user habit, he may wants to get pods at the number of job minAvailable, but he can't get the result for the sum of task replicases is larger than job minAvailable.

Reply:
Sorry I did not explain clearly, if job minavailable is not set, it will be set to sumof(task minavailable), otherwise it will not be changed and keep user settings.

Reply:
> Sorry I did not explain clearly, if job minavailable is not set, it will be set to sumof(task minavailable), otherwise it will not be changed and keep user settings.

That's reasonable.